### DISCUSSION TITLE
sidebar/sanfene/redis

### DISCUSSION THREAD

**Discussion by giscus** [itwanger/tobebetterjavaer-giscus#91] [Announcements]
# sidebar/sanfene/redis

图文详解 53 道Redis面试高频题，这次吊打面试官，我觉得稳了（手动 dog）

https://tobebetterjavaer.com/sidebar/sanfene/redis.html

**Comment by LVXUEYANGTiger** [comment]
已学习

**Comment by NX-207** [comment]
加油

**Comment by hachinekooo** [comment]
2025.1.20

**Comment by MetalGear2021** [comment]
2025.3.17

**Comment by TWoZeroWinner** [comment]
“说一下zset的底层原理？”这一节，“但跳表的缺点是查找只能按顺序进行”，其中跳表应该写错了，按后续有连锁更新的缺点的意思，应该指的是“压缩列表”。请二哥核对一下，此致敬礼。

**Comment by XiaJiayu22** [comment]
底层结构这一块写的太乱了，不能先把Redis数据结构和底层结构对应关系先列出来吗

**Comment by ifbluerain** [comment]
RRateLimiter的实现不是令牌桶，是滑动窗口
